### PR TITLE
Rabbitmq 400

### PR DESCRIPTION
**Pull request by ctdk**
This addresses the orphaned consumer rabbitmq issue, and mangles some graphite-api responses we're putting into the logs in such a way that they don't crash logstash.

### CODE REVIEW

**Comment by Dieterbe** [comment]
mangling is fine with me. if anyone else can confirm the rabbit thing, let's merge this

File: pkg/services/rabbitmq/consumer.go [hunk 1]
```diff
@@ -102,6 +102,11 @@ func (c *Consumer) getChannel() error {
 			log.Printf("connection to rabbitmq lost. %s", er.Error())
 			log.Printf("attempting to create new rabbitmq channel.")
 			err := c.getChannel()
+
+			if c.callback != nil {
```
Comment (woodsaj) [inline pkg/services/rabbitmq/consumer.go]: No!
 I'm not sure how I can be any clearer with this.
  `callback.consume` needs to be called if creating the channel is successful.  So this needs to move to within the below if stanza that checks if `c.getChannel()` worked (no error returned)

Comment (woodsaj) [inline pkg/services/rabbitmq/consumer.go]: lines 106-108 need to move BETWEEN 110 and 111.

```
if err == nil {
  if c.callback != nil {
    c.consume()
  } 
  break
}
```

**Comment by ctdk** [comment]
Moved the consume again. Hopefully I won't be this dense for a while.

**Pull request merged**